src/sentry/static/sentry/app/bootstrap.tsx

```diff
@@ -139,6 +142,11 @@ const render = (Component: React.ComponentType) => {
   }
 };
 
+const RenderPipelineView = (pipelineName: string, props: Object) => {
+  const rootEl = document.getElementById(ROOT_ELEMENT);
+  ReactDOM.render(<Pipeline pipelineName={pipelineName} {...props} />, rootEl);
```

Comment:
@billyvg do we need the try/catch here like we have for the main render function?